### ISSUE TITLE
Play-style intro sometimes doesn't load

### ISSUE THREAD

**Issue by Bgagger** [basstabs/NEForumMessenger#19]
Users where this is known to happen:
Kat
felwin
Momohime00
Rosa Raines

When opening the play style intro. Sometimes the screen is simply blank where the thread text should show.  It can stay for a very long time (30 min) like that with nothing happening.
Often if you go back and try opening it again, it will load on the second attempt. Sometimes it works on the first attempt as well, so does not appear to be consistent.

Some sort of time-out might be good. Where a message is shown that the thread could not load and encourage the user to try again. Or if the app instead tries reloading the data if it times-out without displaying.
As it is now there is no indication that the app is doing something or if something has gone wrong when this happens.

From what I can tell this only happens for players who have more then one post in the play style thread.

**Comment by Bgagger** [comment]
Addition. This also happens for the following players:
AmazingAlexia

Which has only one post in the play-style thread. So either that was a coincidence, or the problem may have several causes. It also appears that once the thread has successfully loaded, subsequent openings will work.

**Comment by basstabs** [comment]
Unable to reproduce this issue. Perhaps a network failure is the issue, wherein the initial call fails. Regardless, I believe redundancy could help here if the initial network call fails. Will look into implementations.

On the plus side, fixed a completely related bug noticed by looking at a user on the above list, so, progress!

**Comment by basstabs** [comment]
Added a redundant network call for each play style intro post: Every  5-10 seconds it resends all requests for play style intro posts that haven't returned successfully yet. If all requests return, the task is cancelled and no more requests are sent.  I also added loading text to the profile page so that it isn't blank while the network calls are being performed. 

I was unable to reproduce the error, so I'm not sure if this fixes it, but it can be tested upon release.
